### PR TITLE
fixing rule - A client made a web request to a potentially harmful file

### PR DESCRIPTION
**Pull request by vakohl**
Required items, please complete
   
   Change(s):
   - fixed regex and variable name, entity mapping, description

   Reason for Change(s):
   - user reported issue - https://github.com/Azure/Azure-Sentinel/issues/8189

   Version Updated:
   - 1.1.2

   Testing Completed:
   - Yes

### CODE REVIEW

File: Detections/ASimWebSession/PotentiallyHarmfulFileTypes.yaml [hunk 1]
```diff
@@ -16,45 +15,63 @@ requiredDataConnectors:
...
+  let default_file_ext_blocklist = dynamic(['.ps1', '.vbs', '.bat', '.scr']); // Update this list as per your requirement
+  let custom_file_ext_blocklist=toscalar(_GetWatchlist('RiskyFileTypes')
+    | extend Extension=column_ifexists("Extension", "")
+    | where isnotempty(Extension)
+    | summarize make_set(Extension)); // If you have an extensive list, you can also create a Watchlist that includes the file extensions you want to detect
   let file_ext_blocklist = array_concat(default_file_ext_blocklist, custom_file_ext_blocklist);
-  _Im_WebSession(url_has_any=file_ext_blocklist, eventresult='Success')
-  | extend requestedFileName=tostring(split(tostring(parse_url(Url)["Path"]),'/')[-1])
-  | extend requestedFileExt=extract(@(\.\w+)$,1,requestedFileName, typeof(string))
+  _Im_WebSession(starttime=ago(10min), url_has_any=file_ext_blocklist, eventresult='Success')
```
Comment (oshezaf) [inline Detections/ASimWebSession/PotentiallyHarmfulFileTypes.yaml:33]: Interesting to understand if and why specifying starttime matters, given that the rule query period is 10 minutes anyways.

**Review by oshezaf — COMMENTED** [review]
No need to change anything but asked a question in the comments.

File: Detections/ASimWebSession/PotentiallyHarmfulFileTypes.yaml (hunk 1 above)
Comment (vakohl) [reply Detections/ASimWebSession/PotentiallyHarmfulFileTypes.yaml:33]: not needed Ofer. As you mentioned, we are specifying query period in rule template.

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**